pom.xml

```diff
@@ -16,38 +16,17 @@
 		<hibernate.version>4.2.7.Final</hibernate.version>
 		<jackson.version>2.3.0</jackson.version>
 		<jetty.version>9.1.1.v20140108</jetty.version>
-		<resteasy.version>3.0.4.Final</resteasy.version>
+		<resteasy.version>3.0.10.Final</resteasy.version>
 		<slf4j.version>1.7.5</slf4j.version>
 	</properties>
 
-    <repositories>
-        <repository>
-            <id>jpacman-release-repository</id>
-            <url>https://github.com/devhub-tud/devhub-mvn-repo/raw/master/releases</url>
-            <releases>
-                <enabled>true</enabled>
-                <updatePolicy>always</updatePolicy>
-            </releases>
-            <snapshots>
-                <enabled>false</enabled>
-            </snapshots>
-        </repository>
-
-        <repository>
-            <id>jpacman-snapshot-repository</id>
-            <url>https://github.com/devhub-tud/devhub-mvn-repo/raw/master/snapshots</url>
-            <releases>
-                <enabled>false</enabled>
-            </releases>
-            <snapshots>
-                <enabled>true</enabled>
-                <updatePolicy>always</updatePolicy>
-            </snapshots>
-        </repository>
-    </repositories>
-
 	<dependencies>
 		<dependency>
+			<groupId>javax.ejb</groupId>
```

Comment:
I believe Jetty accidentally removed it from it's dependencies. We do not use EJB in our own code, but when you remove the dependency, the server fails to start.